agent/handlers/v1_handlers_test.go

```diff
@@ -78,6 +78,20 @@ func TestGetTaskByDockerID(t *testing.T) {
 	taskDiffHelper(t, []*api.Task{testTasks[1]}, TasksResponse{Tasks: []*TaskResponse{&taskResponse}})
 }
 
+func TestGetTaskByShortDockerID(t *testing.T) {
+	// stateSetupHelper uses the convention of dockerid-$arn-$containerName; the
+	// first task has a container name prefix of dockerid-tas
+	recorder := performMockRequest(t, "/v1/tasks?dockerid=dockerid-tas")
+
+	var taskResponse TaskResponse
+	err := json.Unmarshal(recorder.Body.Bytes(), &taskResponse)
+	if err != nil {
```

Comment:
Can you change this to `require.NoError(t, err)`?  You'll need to add an import for `github.com/stretchr/testify/require`.  We're trying to slowly change to using the `assert` and `require` packages and new tests should use those.